services/core-web/src/tests/components/mine/NoticeOfWork/__snapshots__/MineNoticeOfWorkTable.spec.js.snap

```diff
@@ -62,6 +62,7 @@ exports[`MineNoticeOfWorkTable renders properly 1`] = `
     Array [
       Object {
         "document": Object {},
+        "is_historic": undefined,
```

Comment:
I think you are just missing this value in the mocked data.

Reply:
updated!